src/components/landingPage/LandingPage.js

```diff
@@ -1,172 +1,203 @@
-    },
+  userStories: {
+    backgroundColor: "rgba(58, 83, 155, 0.9)",
+    display: "inline-block",
+  },
 
-    paper: {
-        height: 350,
-        width: 250,
-        margin: theme.spacing(2),  
-        paddingTop: '10%',
-        textAlign: 'center'
-    },
+  paper: {
+    height: 350,
+    width: 250,
+    margin: theme.spacing(2),
+    paddingTop: "10%",
+    textAlign: "center",
+  },
 
-    userStoriesTitle: {
-        textAlign: 'center',
-        fontSize: '2em',
-    },
+  userStoriesTitle: {
+    textAlign: "center",
+    fontSize: "2em",
+  },
 
-    users: {
-        borderRadius: '50%',
-        height: 90,
-        display: 'block',
-        marginLeft: 'auto',
-        marginRight: 'auto',
-    },
+  users: {
+    borderRadius: "50%",
+    height: 90,
+    display: "block",
+    marginLeft: "auto",
+    marginRight: "auto",
+  },
 
-    userReview: {
-        margin: '5% 10% 0 10%',
-    },
-}))
+  userReview: {
+    margin: "5% 10% 0 10%",
+  },
+}));
 
 export default function LandingPage() {
-    const classes = useStyles();
+  const classes = useStyles();
 
-    return (
-        <div className="full-container">
-            <div className="top-container">
-                <div className="top-container-info">
-                    <h1>Get help with your grant today!</h1>
-                    <h3>An online platform that connects applicants and organizations with skilled writers eager to help with your grant proposals.</h3>
-                    <div className="buttons">
-                        <Button 
-                            variant="contained"
-                            color="primary"
-                            className={classes.buttons}
-                            size="large"
-                        >
-                            Register
-                        </Button>
-                        <Button 
-                            variant="contained"
-                            color="primary"
-                            className={classes.buttons}
-                            size="large"
-                        >
-                            Login
-                        </Button>
-                    </div>
-                </div>
-            </div>
-            <Grid 
-                container 
-                direction="column" 
-                className={classes.info}
+  return (
+    <div className="full-container">
+      <div className="top-container">
+        <div className="top-container-info">
+          <h1>Get help with your grant today!</h1>
+          <h3>
+            An online platform that connects applicants and organizations with
+            skilled writers eager to help with your grant proposals.
+          </h3>
+          <div className="buttons">
+            <Button
+              variant="contained"
+              color="primary"
+              className={classes.buttons}
+              size="large"
+              component={Link}
```

Comment:
took me forever to find it, gotta love those prettier updated that github wants to make sure we don't miss 🙃